zivid_camera/src/zivid_camera.cpp

```diff
@@ -33,58 +65,139 @@ sensor_msgs::PointField createPointField(std::string name, uint32_t offset, uint
 
 bool big_endian()
 {
-  return BOOST_ENDIAN_BIG_BYTE;
+  union {
```

Comment:
could you change this method to camel-case bigEndian() to match the other funcitons?

Reply:
df24da9329e235dcb30399b53eb6c1781f73d634